src/common.rs

```diff
@@ -77,7 +83,7 @@ pub struct Real {
 #[derive(Clone, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
 pub enum Literal {
     Null,
-    Integer(i64),
+    Integer(i64), // todo should this be a i128 then?
```

Comment:
… or should we have a signed vs. unsigned representation for literals too?

Reply:
I think having variants of `Literal` is a good plan -- it avoids typecasting in external code producing/consuming `Literals`.

That said, `Literals` are often created from parts of the MySQL query string, where signedness of integers may be ambiguous. I guess we could go for unsigned unless the literal actually has a negative sign in front of it? (side note: I'm not sure the parse rules for literals actually cover negative numbers at the moment.)